kuma/javascript/src/ssr.jsx

```diff
@@ -9,8 +9,20 @@ import { localize } from './l10n.js';
  * a JSON object of document data. It is used by ../ssr-server.js
  */
 export default function ssr(data) {
-    // Store the string catalog so that l10n.gettext() can do translations
-    localize(data.requestData.locale, data.localizationData);
+    // Before we can render the App, we need to call localize() so that
+    // gettext() and ngettext() will work property during rendering.
+    let pluralFunction = null;
+    if (data.pluralExpression) {
+        // Creating a function like this in client-side JS will usually
+        // violate CSP. But we're running in Node here, so it is okay.
+        pluralFunction = new Function(
+            'n',
+            `let v=(${
+                data.pluralExpression
+            });return(v===true)?1:((v===false)?0:v);`
+        );
```

Comment:
During server-side rendering, we need to build the plural function explicitly, but we're running in Node, so we don't have to worry about CSP in this case.